e2e_test/batch/aggregate/with_over_window.slt.part

```diff
@@ -0,0 +1,82 @@
+statement ok
+create table t (a int, b int, c int, d int, e int);
+
+statement ok
+insert into t values
+    (1, 23, 84, 11, 87),
+    (2, 34, 29, 22, 98),
+    (3, 45, 43, 33, 10),
+    (4, 56, 83, 44, 26),
+    (5, 68, 20, 55, 12),
+    (5, 68, 90, 66, 34),
+    (5, 68, 11, 77, 32);
+
+query II
+select
+    a,
+    sum((sum(b))) over (partition by a order by a)
+from t
+group by a;
```

Comment:
Better to add `order by a` to ensure consistent result.

Reply:
If it is for consistent result, should `order by` constraints be added to every query?

Reply:
Yes. Should be added to every query in this file.